app/components/filters/filter.ts

```diff
@@ -3,17 +3,19 @@ import RouterService from '@ember/routing/router-service';
 import { service } from '@ember/service';
 import { get } from '@ember/object';
 
-interface ArgsInterface {
-  id: string;
-  queryParam: string;
-  queryParamsA: string;
-  queryParamsB: string;
+interface Signature {
```

Comment:
I updated this to match the setup in the ember-cli-ts documentation: https://docs.ember-cli-typescript.com/ember/components